hardware/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,486 @@
+  <img src="files/Layout-Design-Guide/crystal3.png">
+</p>
+
+<div align="center">
+
+<b>Figure 20.</b> 
+
+</div>
+
+****
+## 3.4. Layout Design Practices - VDD filtering
+
+1. The lowest value capacitors have to be placed the closest to the VDD pins. Beside that, they need good grounding (with many close vias) at the ground side. The largest value capacitor filters can be placed far away from the VDD pins and is not required in case of battery operation.
+
+
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/VDD.png">
+</p>
+
+<div align="center">
+
+<b>Figure 21.</b> 
+
+</div>
+
+***
+
+## 3.5. Layout Design Practices - General
+
+1. Use 50 Ohm grounded coplanar transmission lines to connect distant RF components. Online impedance calculators can be used to get the necessary trace width or see below the typical values for 50 Ohm grounded coplanar lines. 
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/coplanar2.png">
+</p>
+
+<div align="center">
+
+<b>Figure 22.</b> 
+
+</div>
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/coplanar.png">
+</p>
+
+<div align="center">
+
+<b>Figure 23.</b> 
+
+</div>
+
+2. The grounding effect in the thermal straps used with capacitors should be improved. In addition, the trace near the GND pin of these capacitors should be thickened, this will minimize series parasitic inductance between the ground pour and the GND pins. Additional vias placed close to the GND pin of capacitors (thus connecting it to the bottom or inner layer GND plane) will further help reduce these effects.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/vias.png">
+</p>
+
+<div align="center">
+
+<b>Figure 24.</b> 
+
+</div>
+
+3. The exposed pad footprint for the paddle of the EFR32 IC should use as many vias as possible to ensure good grounding and heat sink capability. In the reference designs there are 25 vias for the 7x7 mm sized package ICs, each with 10 mil diameter. The paddle ground should also be connected to the top layer GND metal, if possible, to further improve RF grounding; this may be accomplished with diagonal trace connections through the corners of the EFR32 IC footprint.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/vias2.png">
+</p>
+
+<div align="center">
+
+<b>Figure 25.</b> 
+
+</div>
+
+4. Use as many grounding vias at the GND metal edges (especially at the edge of the PCB and along the VDD traces) as possible in order to reduce their harmonic radiation caused by the fringing field.
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/vias3.png">
+</p>
+
+<div align="center">
+
+<b>Figure 26.</b> 
+
+</div>
+
+
+5. In a design with more than two layers, all of the wires/traces should be placed in one of the inner layers, especially the VDD trace. Also avoid putting it close to the edge of the PCB. The whole top and bottom layers should contain as much continuous GND metallization as possible in order to reduce the traces' radiations.
```

Comment:
The whole top and bottom layers should contain as much continuous GND metallization as possible in order to reduce the `harmonics/spurs radiations from the signal traces`.